### DISCUSSION TITLE
Use variable as the choice of filter

### DISCUSSION THREAD

**Discussion by GeNTooGeek** [WarmUpTill/SceneSwitcher#725] [Ideas]
Hello,

When I select the Filter macro, I am able to select a variable in the Scene selector, but I am not able to do so in the Filter selector.  Would it be possible to add that?

**Comment by WarmUpTill** [comment]
Thanks for the suggestion!

That should certainly be doable - it is just something I have not gotten around to yet. :)

**Reply by WarmUpTill** [reply]
A build containing this functionality should be available here in a few minutes:
https://github.com/WarmUpTill/SceneSwitcher/actions/runs/4810084096?pr=729

Unfortunately I did not get much of a chance to test it.
So please report if you run into any issues!
